packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -93,6 +93,7 @@ export default function NavigationInnerBlocks( {
 			onInput,
 			onChange,
 			allowedBlocks: ALLOWED_BLOCKS,
+			inserterPriority: INSERTER_PRIORITY,
```

Comment:
I think a different name more close to what it actually is, would be better. Maybe `prioritizedInserterBlocks` or something like that..

Reply:
Ok no problem.